packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -934,3 +969,28 @@ enum ContainerInsights {
    */
   DISABLED = 'disabled',
 }
+
+/**
+ * A Capacity Provider strategy to use for the service.
+ */
+export interface CapacityProviderStrategy {
+  /**
+   * The name of the Capacity Provider. Currently only FARGATE and FARGATE_SPOT are supported.
+   */
+  readonly capacityProvider: string;
+
+  /**
+   * The base value designates how many tasks, at a minimum, to run on the specified capacity provider. Only one
+   * capacity provider in a capacity provider strategy can have a base defined. If no value is specified, the default
+   * value of 0 is used.
+   *
+   * @default - none
+   */
+  readonly base?: number;
+
+  /**
+   * The weight value designates the relative percentage of the total number of tasks launched that should use the specified
+capacity provider. The weight value is taken into consideration after the base value, if defined, is satisfied.
+   */
+  readonly weight: number;
```

Comment:
Should weight be an optional property? For instance if they want to use `Fargate` as the only Capacity Provider, they should be able to pass it in without any weight and it should default to 0 (according to the [cfn docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-service-capacityproviderstrategyitem.html#cfn-ecs-service-capacityproviderstrategyitem-weight))